src/nix/repl.cc

```diff
@@ -654,7 +663,10 @@ void NixRepl::reloadFiles()
         if (!first) notice("");
         first = false;
         notice("Loading '%1%'...", i);
-        loadFile(i);
+
+        settings.isExperimentalFeatureEnabled(Xp::Flakes)
+        ? loadFlake(i)
+        : loadFile(i);
```

Comment:
```suggestion
        if (settings.isExperimentalFeatureEnabled(Xp::Flakes))
            loadFlake(i);
        else
            loadFile(i);
```
since it's not really idiomatic in C++ (IMHO) to have the conditional operator at statement level.